gcal/widget.go

```diff
@@ -158,7 +173,7 @@ func (widget *Widget) eventTimestamp(event *calendar.Event) string {
 		return startTime.Format(wtf.FriendlyDateFormat)
```

Comment:
One caveat - I'm not sure what change we should make for this branch of the conditional. I suspect we don't want the current behavior (there's already a date heading, so printing the date is redundant, and now won't fit as well on the same line as the title), but this branch doesn't actually trigger for any events in my calendar so I'm not sure what the use case is (all-day events, maybe?) or what this _should_ look like in the new format.

Reply:
Pretty sure that's for all-day events. I'll try this out and see how it looks with those.